### DISCUSSION TITLE
SOLOMINER v2.0.beta1

### DISCUSSION THREAD

**Discussion by Danaka-dev** [Danaka-dev/solominer2#2] [Announcements]
**Solominer v2.0.BETA 1**
*(early access)*

***features***

+ All the information to make the right mining decision at your fingertip\
\
    *How many blocks you will mine per day for each blockchain, what will be the earning per day, in $, in
BTC... it's all there.*

+ Mining directly on your local node in total solo mode\
\
*No fee, no centralization, all the fun. Several cpu mineable blockchains supported, use your existing own core 
daemon/wallet or the ones provided, all fully configurable from the comfort of the user interface.*

+ A nifty dashboard and mining monitor\
\
*Giving you a clear and concise view of your ongoing mining journey. Hashrate, part per second, luck, progress 
toward statistical completion of a block, block mined... all in a nice interface with a cool retro feel (hope you like
it as much as we do :)*

+ Switch between the digital currency you are mining with a click of a button \
\
*Start and stop core daemon or wallet application from within the dashboard, get a new address, select or switch the 
currency you are mining with a click or let solominer auto mode select the blockchain with the highest earnings
for you automatically.*   

<hr /><em>This discussion was created from the release <a href='https://github.com/Danaka-dev/solominer2/releases/tag/v2.0.beta1'>SOLOMINER v2.0.beta1</a>.</em>

**Comment by teqteq14** [comment]
Bonjour,
je souhaite justement miner en solo ! Comment cela foncionne ? quel os (windows, linux...).
Comment le mettre en place ?
merci
